### DISCUSSION TITLE
Getting output of bpf_printk

### DISCUSSION THREAD

**Discussion by Monk3yDLuffy** [microsoft/ebpf-for-windows#1291] [Q&A] [answered]
Hi,
I was trying to print bpf_printk output using Instructions provided in [https://github.com/microsoft/ebpf-for-windows/blob/main/docs/GettingStarted.md](url), i tried using wpr file but got
`C:\Windows\System32>wpr.exe -start ebpfforwindows.wprp -filemode

        The system cannot find the file specified.

        Error code: 0x80070002`
I also tried creating a new ebpfforwindows.wprp file but got
`C:\Windows\System32>fsutil file createnew ebpfforwindows.wprp 0
File C:\Windows\System32\ebpfforwindows.wprp is created

C:\Windows\System32>wpr.exe -start ebpfforwindows.wprp -filemode

        Error code: 0xc00ce558
        XML document must have a top level element.

        Line Number: 0
        Column Number: 0`

I also added `<?xml version="1.0" encoding="utf-8" standalone='yes'?>`
in the file still got the following .
can u pls help me with it

**Comment by dthaler** [comment, marked as answer]
The wprp file is part of the distribution or you can grab it from https://github.com/microsoft/ebpf-for-windows/blob/main/scripts/ebpfforwindows.wprp
We can update the GettingStarted.md doc to make this more clear.

Which method did you install eBPF for Windows?  If you used the MSI (i.e., Method 1 in the install instructions), then the wprp file should be in %ProgramFiles%\ebpf-for-windows\ebpfforwindows.wprp

**Reply by Monk3yDLuffy** [reply]
thanks it is working now
